crates/services/txpool_v2/src/service.rs

```diff
@@ -264,6 +263,15 @@ where
                 }
             }
 
+            new_status_update = self.subscriptions.all_tx_status_updates.recv() => {
+                if let Ok((tx_id, tx_status)) = new_status_update {
+                    self.process_tx_update(tx_id, tx_status);
+                    TaskNextAction::Continue
+                } else {
+                    TaskNextAction::Stop
+                }
+            }
```

Comment:
Why do we want to subscribe in the TxPoolService instead of TxPoolWorker? Looks like redundant hop

Reply:
As explained in PR description the goal was to subscribe in the TxPoolService to all events and filter only the useful ones (for now the perconfirmaiton) that will be sent to the TxPoolWorker.

Reply:
But if you do subscription only for pre confirmations, then filtering will be done by `TxStatusManager`=)

Reply:
Yeah yeah but it's not very future-prone but ok will do tomorrow

Reply:
Done